src/main/java/ee/tuleva/onboarding/mandate/application/Application.java

```diff
@@ -17,6 +18,16 @@
   private ApplicationStatus status;
   private ApplicationDetails details;
 
+  @JsonIgnore
+  public boolean isPending() {
+    return status.equals(ApplicationStatus.PENDING);
+  }
+
+  @JsonIgnore
```

Comment:
A good reason to extract an `ApplicationDto` - so we wouldn't accidentally leak this kind of methods thru the REST API.